cmd/ddev/cmd/composer_test.go

```diff
@@ -57,7 +57,6 @@ func TestComposerCmdCreateConfigInstall(t *testing.T) {
 		assert.NoError(err)
 
 		t.Cleanup(func() {
-			//nolint: errcheck
 			err = app.Stop(true, false)
```

Comment:
I suspect this originally had an errcheck and at some point the assert got added. But these days as you know I have gone to `_ = ...` where I really don't care about the error.

Reply:
This was a manual change, I went through existing `// nolint:` comments, removed them all, and then ran `golangci-lint run` and put back half of them.